README.md

```diff
@@ -0,0 +1,15 @@
+# fec-infrastructure
+
+Terraform configuration and Concourse pipeline for managing custom FEC infrastructure in AWS.
+
+## Setup
+* Concourse pipeline is deployed to a concourse team owned by 18F Infrastructure
+* Terraform configuration is deployed via Concourse to an AWS account owned by Infrastructure
+* FEC staff are responsible for adding resources that incur a cost to Chandika
+
+## Workflow
+* FEC team member sends a pull request to 18F/fec-infrastructure
+* If applicable, FEC team member sends updated Concourse credentials to infrastructure staff
+* 18F Infrastructure team member reviews pull request to ensure that all resources are properly tagged
+* After pull request is merged, updates are automatically deployed via Concourse
+* If applicable, Infrastructure team member sends updated Terraform outputs to FEC staff
```

Comment:
Okay, I see now (sorry, only exposure to Chandika currently is what @mogul showed us briefly when we spoke last week).  And yes, we'd definitely need the endpoints for the apps too. :-)  Thanks!